cmd/mattermost/commands/config.go

```diff
@@ -102,3 +117,124 @@ func configSubpathCmdF(command *cobra.Command, args []string) error {
 
 	return nil
 }
+
+func configGetCmdF(command *cobra.Command, args []string) error {
+	app, err := InitDBCommandContextCobra(command)
+	if err != nil {
+		return err
+	}
+	defer app.Shutdown()
+
+	// create the model for config
+	// Note: app.Config() returns a pointer, make appropriate changes
+	config := app.Config()
+
+	// get the print config setting and any error if there is
+	out, err := printConfigValues(configToMap(*config), strings.Split(args[0], "."), args[0])
+	if err != nil {
+		return err
+	}
+
+	fmt.Printf("%s", out)
+
+	return nil
+}
+
+// printConfigValues function prints out the value of the configSettings working recursively or
+// gives an error if config setting is not in the file.
+func printConfigValues(configMap map[string]interface{}, configSetting []string, name string) (string, error) {
+
+	out := &bytes.Buffer{}
+
+	res, ok := configMap[configSetting[0]]
+	if !ok {
+		return "", fmt.Errorf("%s configuration setting is not in the file", name)
+	}
+	value := reflect.ValueOf(res)
+	switch value.Kind() {
+	case reflect.Map:
+		if len(configSetting) == 1 {
+			fmt.Fprintf(out, "%s", printMap(value, 0))
+			return out.String(), nil
+		}
+		return printConfigValues(res.(map[string]interface{}), configSetting[1:], name)
+	default:
+		if len(configSetting) == 1 {
+			fmt.Fprintf(out, "%s: \"%v\"\n", name, res)
```

Comment:
Probably you can change this two lines to something like `return fmt.Sprintf("%s: \"%v\"\n", name, res)` and remove the out variable entirely.

Reply:
Done. Pushed the changes.